src/Descriptors/Actions/ActionDescriptor.php

```diff
@@ -80,7 +81,13 @@ public function action(): Operation
      */
     protected function description(): string
     {
-        return '';
+        if (!$this->route->schema() instanceof DescribesEndpoints) {
```

Comment:
Maybe make this a var `$schema`? This way you only call `$this->route->schema()` once.

Reply:
I could do that, but it results in code that in my opinion looks a bit iffy. I'd rather call the schema method once more to make the code look a more readable. Especially when the schema method is only a getter.

Reply:
I agree that it would be easier to change, but in this example I think it looks weird due to the instance of check.

```php
    /**
     * @return string
     */
    protected function description(): string
    {
        /** @var DescribesEndpoints $schema */
        $schema = $this->route->schema();

        if (!$schema instanceof DescribesEndpoints) {
            return '';
        }

        return $schema->describeEndpoint($this->route->route()->getName());
    }
 ```

But I can change it into this if this is what you like more.

Reply:
Yes please, :)